src/mixins/filing-mixin.ts

```diff
@@ -132,7 +132,7 @@ export default class FilingMixin extends Vue {
       },
       dissolution: {
         custodialOffice: this.getRegisteredOfficeAddress,
-        dissolutionType: DissolutionTypes.VOLUNTARY // To be applied dynamically when we have dissolution variations.
+        dissolutionType: DissolutionTypes.HDV // To be applied dynamically when we have dissolution variations.
```

Comment:
I'm not 100% sure what Create UI will do with this draft data atm. We need to work this through...

Reply:
I just checked, and Create UI doesn't yet do anything with dissolution type in the draft filing, so the change above shouldn't break anything.

The next dev to work on this will have to sync up Create UI with Filings UI, however.